rpcserver.go

```diff
@@ -957,8 +957,25 @@ func (r *rpcServer) ListAssets(ctx context.Context,
 		return nil, err
 	}
 
+	// We will also report the number of unconfirmed transfers. This is
+	// useful for clients as unconfirmed asset coins are not included in the
+	// asset list.
+	transfersResp, err := r.ListTransfers(
```

Comment:
By default, `ListTransfers` returns both confirmed and unconfirmed transfers.
So I think we should instead call into `r.cfg.AssetStore.QueryParcels(ctx, nil, true)` to get the unconfirmed transfers directly.